src/common/components/layouts/pages/settings/Settings.js

```diff
@@ -10,6 +10,7 @@ import PageNavigation from '../../../shared/pageNavigation/PageNavigation';
 import { getServerGeneralStatus } from '../../../../store/serverInfo/selectors';
```

Comment:
Page reload on Inspect page results in redirect to File upload page, but we should stay on the same page if file hasn't changed (not file name, but the file itself). For example, you can save the upload timestamp or generate some random hash on upload in order to compare them later  and check if file has really changed